WirelessModules/src/boot.py

```diff
@@ -0,0 +1 @@
+# boot.py - - runs on boot-up
```

Comment:
Don't have an ESP32 on me to test for myself - if this file runs on boot, `main.py` won't ever be called, right? Or I'm missing something

Reply:
This is an extra script that gets run before main.py. We can ignore it for now